HACKING.md

```diff
@@ -26,6 +28,46 @@ And lastly get Webpack and the development dependencies:
     $ npm install
     $ ./node_modules/bower/bin/bower install
 
+When relying on CI to run the test suite, this is all that is
+necessary to work on the JavaScript components of Cockpit.
+
+To actually build the Cockpit binaries themselves from source
+(including to run the integration tests locally), you will need
+additional header files and other components. Check
+`tools/cockpit.spec` for the concrete Fedora build dependencies.
+The following should work in a fresh Git clone:
+
+    $ sudo yum-builddep tools/cockpit.spec
```

Comment:
This is missing node and npm; that part got removed below, but not re-added here.